lib/cli/options.js

```diff
@@ -110,15 +110,19 @@ const parse = (args = [], defaultValues = {}, ...configObjects) => {
   // 4. we can then reapply the values after yargs-parser is done.
   const nodeArgs = (Array.isArray(args) ? args : args.split(' ')).reduce(
     (acc, arg) => {
-      const pair = arg.split('=');
-      let flag = pair[0];
-      if (isNodeFlag(flag, false)) {
-        flag = flag.replace(/^--?/, '');
-        return arg.includes('=')
-          ? acc.concat([[flag, pair[1]]])
-          : acc.concat([[flag, true]]);
+      try {
```

Comment:
[Refactor] `try...catch` is a very blunt tool that addresses a symptom, not a root cause of something being wrong. I think a more direct fix would be to check that the type of the arg is what we expect: a string.

I wrote up an explainer here with examples: https://www.joshuakgoldberg.com/blog/try-catch-as-little-as-possible